tests/integration/bootstrap.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Yoast\WPTestUtils\WPIntegration;
+
+require_once dirname( dirname( __DIR__ ) ) . '/vendor/yoast/wp-test-utils/src/WPIntegration/bootstrap-functions.php';
+
+$_tests_dir = getenv( 'WP_TESTS_DIR' );
+
+if ( ! $_tests_dir ) {
+	$_tests_dir = '/tmp/wordpress-tests-lib';
+}
+
+// Give access to tests_add_filter() function.
+require_once $_tests_dir . '/includes/functions.php';
+
+function _manually_load_plugin() {
+	require dirname( dirname( __DIR__ ) ) . '/wpcom-legacy-redirector.php';
+}
+tests_add_filter( 'muplugins_loaded', '_manually_load_plugin' );
+
+// Make sure the Composer autoload file has been generated.
+WPIntegration\check_composer_autoload_exists();
+// Load WordPress.
+require $_tests_dir . '/includes/bootstrap.php';
+/*
+ * Register the custom autoloader to overload the PHPUnit MockObject classes when running on PHP 8.
+ *
+ * This function has to be called _last_, after the WP test bootstrap to make sure it registers
+ * itself in FRONT of the Composer autoload (which also prepends itself to the autoload queue).
+ */
+WPIntegration\register_mockobject_autoloader();
```

Comment:
The [WP-Test-Utils docs](https://github.com/Yoast/wp-test-utils#bootstrap-utility-functions-and-custom-autoloader) suggest that these three calls could be replaced with `WPIntegration\bootstrap_it();` which is defined [here](https://github.com/Yoast/wp-test-utils/blob/c5cdabd58f2aa6f2a93f734cf48125f880c90101/src/WPIntegration/bootstrap-functions.php#L94-L108).

However, I couldn't get `WPIntegration\get_path_to_wp_test_dir()` (defined [here](https://github.com/Yoast/wp-test-utils/blob/c5cdabd58f2aa6f2a93f734cf48125f880c90101/src/WPIntegration/bootstrap-functions.php#L32)) which is [called](https://github.com/Yoast/wp-test-utils/blob/c5cdabd58f2aa6f2a93f734cf48125f880c90101/src/WPIntegration/bootstrap-functions.php#L135) at the start of `load_wp_test_bootstrap()`, to return anything other than false. 

`get_path_to_wp_test_dir()` contains various checks, and even with defining `WP_TESTS_DIR` and `WP_DEVELOP_DIR` via `<php><env.../></php>` in the config file, I couldn't get it working.

May circle back to this one in the future. cc @jrfnl for inspiration 🙏🏻

Reply:
@GaryJones Please report the issue (with enough detail to reproduce) in the WP Test Utils repo. If nothing else, debugging it in an issue there could possibly also help other users.

Reply:
Also: when `WPIntegration\get_path_to_wp_test_dir()` doesn't work, the autoloader to get the WP versions of the MockObject classes won't work and will defer to the Composer installed version of those classes (the PHPUnit 7.5 ones), which will break on PHP 8.

If no mocking is used in any of the tests, this will go unnoticed.

See: https://github.com/Yoast/wp-test-utils/blob/c5cdabd58f2aa6f2a93f734cf48125f880c90101/src/WPIntegration/Autoload.php#L63